operator/apis/platform/v1alpha1/common_types.go

```diff
@@ -5,6 +5,7 @@ package v1alpha1
 import (
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/pointer"
```

Comment:
According to https://pkg.go.dev/k8s.io/utils/pointer it is deprecated.

Reply:
Ok, how about creating a separate ticket for getting rid of deprecated dependencies such as these?